bin/run-transcrypt.sh

```diff
@@ -0,0 +1,7 @@
+# In $SOURCE_VERSION is set, we are deploying on Scalingo
+# On scalingo, we don't have a git repo, so decrypting is handled
+# differently in `buildpack-run.sh`
+if [ -z "$SOURCE_VERSION" ]
+then
+  transcrypt -y -c aes-256-cbc -p "$TRANSCRYPT_KEY"
```

Comment:
Shouldn't we call `bin/run-transcrypt.sh` here?

Reply:
It is the content of `bin/run-transcrypt.sh` itself 😁

Reply:
Yeah the idea is to have one and only one single place to update the command if needed

Reply:
hmmm sure, but you are asking me to modify the content of `bin/run-transcrypt.sh` itself here, or did I miss something?

Reply:
Bah, I commented that on the wrong file ^^ the idea is to have a single place where we define the caommd line call ^^